packages/web/config/chain-infos.ts

```diff
@@ -2346,6 +2346,30 @@ const chainInfos = (
       features: ["ibc-transfer", "ibc-go"],
       explorerUrlToTx: "https://explorer.thesilverfox.pro/beezee/tx/{txHash}",
     },
+    {
+      rpc: "https://rpc.medas-digital.io:26657",
+      rest: "https://lcd.medas-digital.io:1317",
+      chainId: "medasdigital-1",
+      chainName: "medasdigital",
+      bip44: {
+        coinType: 118,
+      },
+      bech32Config: Bech32Address.defaultBech32Config("medas"),
+      currencies: [
+        {
+          coinDenom: "MEDAS",
+          coinMinimalDenom: "umedas",
+          coinDecimals: 6,
+          // coinGeckoId: "medasdigital",
+          coinGeckoId: "pool:umedas",
```

Comment:
Shall we use pool 859 to define this 'id' in the prices.ts file? With this, the MEDAS token will have a price to display in the frontend.

Reply:
Yes please

Reply:
Ok, added to price.ts